tests/virt_autotest/libvirt_virtual_network_init.pm

```diff
@@ -50,7 +51,8 @@ sub run_test {
     my ($ACTIVE_POOL_NAME, $AVAILABLE_POOL_SIZE) = $self->get_active_pool_and_available_space();
     record_info('Detect Active POOL NAME:',    $ACTIVE_POOL_NAME);
     record_info('Detect Available POOL SIZE:', $AVAILABLE_POOL_SIZE . 'GiB');
-    assert_script_run("test $AVAILABLE_POOL_SIZE -ge 60", fail_message => "The SUT needs at least 60GiB available space of active pool for virtual network test");
+    my $expected_pool_size = (is_xen_host) ? '60' : '40';
```

Comment:
What's the actual required disk space before guest installation, and after installation but during virtual_network_init step? They should be different actually, because the installed guests take libvirt/image space a lot, so the value before guest installation should be bigger. In https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11498/files, which checks available disk space before guest installation, the same check value is used as this PR which checks after guest installation. So maybe need another value here.

Reply:
Good point ! 
sure, PR https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11498 had check with disk space for /var/lib/libvirt before guest installation  with `my $expected_libvirt_size = (is_xen_host) ? '60' : '40'; `
Also, we had **defined** (not used ) qcow2 file with 20GiB disk space for each guest installation.
yes, we should follow check with disk space and define qcow2 file to define expected pool size here
for example `my $expected_pool_size = (is_xen_host) ? '40' : '20';` define the enough pool size under /var/lib/libvirt to clone source guest during virtual routed network test.

Reply:
@guoxuguang I re-checked the code, why the new value is even smaller? The expected_pool_size should be bigger than "the needed disk size for installing guest + additional disk size for virtual network test to clone guest". So it should be BIGGER SIZE before installing guests. And our tests/qam tests uses different guest disk size too, which need different value too.

Reply:
Great,  imho,  expected_pool_size should be **the deployed guest images(at least KVM-20GiB, XEN-40GiB) + additional disk size(at least 20GiB) for virtual network test to clone guest**, which should be minimum value. if detect current pool value < this expected_pool_size (minimum value), will call` fail_message => "The SUT needs at least " . $expected_pool_size . "GiB available space of active pool for virtual network test"`
So, there was not used with the BIGGER SIZE before our virt_auto test.

Reply:
Also, regarding qam test, refer to https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/virtualization/universal/prepare_guests.pm, they were used with /var/lib/libvirt/images/xen/ to restore guest images different with us.
```
    assert_script_run "mkdir -p /var/lib/libvirt/images/xen/";

    if (is_sle('<=12-SP1')) {
        script_run "umount /home";
        assert_script_run qq(sed -i 's/\\/home/\\/var\\/lib\\/libvirt\\/images\\/xen/g' /etc/fstab);
        script_run "mount /var/lib/libvirt/images/xen/";
    }
```

Reply:
> @guoxuguang I re-checked the code, why the new value is even smaller? The expected_pool_size should be bigger than "the needed disk size for installing guest + additional disk size for virtual network test to clone guest". So it should be BIGGER SIZE before installing guests. And our tests/qam tests uses different guest disk size too, which need different value too.

@guoxuguang Sorry, I was wrong on some part. This PR should use SMALLER SIZE indeed because it is a step executed after guest installation , but before virtual nework test. Please set proper value based on ours and qam's different guest size.